extensions/wikia/MyHome/ActivityFeedHelper.php

```diff
@@ -262,7 +262,6 @@ function CommunityWidgetAjax() {
 	global $wgRequest, $wgLang, $wgLanguageCode, $wgContentNamespaces;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)